tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/ProcessStart/ProcessStartIntegration.cs

```diff
@@ -0,0 +1,68 @@
+// <copyright file="ProcessStartIntegration.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.ComponentModel;
+using System.Diagnostics;
+using Datadog.Trace.ClrProfiler.CallTarget;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.Process
+{
+    /// <summary>
+    /// System.Net.Http.HttpClientHandler calltarget instrumentation
+    /// </summary>
+    [InstrumentMethod(
+   AssemblyName = "System",
+   TypeName = "System.Diagnostics.Process",
+   MethodName = "Start",
+   ReturnTypeName = ClrNames.Process,
+   MinimumVersion = "1.0.0",
+   MaximumVersion = "7.*.*",
+   IntegrationName = nameof(Configuration.IntegrationId.ProcessStart))]
+    [InstrumentMethod(
+   AssemblyName = "System.Diagnostics.Process",
+   TypeName = "System.Diagnostics.Process",
+   MethodName = "Start",
+   ReturnTypeName = ClrNames.Process,
+   MinimumVersion = "1.0.0",
+   MaximumVersion = "7.*.*",
+   IntegrationName = nameof(Configuration.IntegrationId.ProcessStart))]
```

Comment:
nit: you can simplify this with `AssemblyNames`:
```suggestion
    [InstrumentMethod(
       AssemblyNames = new [] { "System", "System.Diagnostics.Process" },
       TypeName = "System.Diagnostics.Process",
       MethodName = "Start",
       ReturnTypeName = ClrNames.Process,
       MinimumVersion = "1.0.0",
       MaximumVersion = "7.*.*",
       IntegrationName = nameof(Configuration.IntegrationId.ProcessStart))]
```

Reply:
Cool, thanks. I will do it like that.